docs/quickstart-gke.md

```diff
@@ -34,11 +34,11 @@ In this section, you will deploy an Google Kubernetes Engine (GKE) cluster using
   - `GCLOUD_PROJECT` specifies the Google Cloud project. `gcloud projects list` lists your Google Cloud projects.
   - `GCLOUD_REGION` specifies the Google Cloud region. `gcloud compute regions list` lists the available Google Cloud regions.
   - `GCLOUD_DNS_ZONE` specifies the DNS suffix for the externally-visible websites and services deployed in the cluster.
-  - `GCLOUD_AUTHZ_DOMAIN` specifies the email domain of authorized users and needs to be a [G Suite](https://gsuite.google.com/) domain. Alternatively you could specify the value `*`, but be __WARNED__ that this will authorize any Google account holder to authenticate with OAuth to your cluster services.
+  - `GCLOUD_AUTHZ_DOMAIN` specifies the email domain of authorized users and needs to be a [G Suite](https://gsuite.google.com/) domain. Alternatively, you could specify the value `*`, but be __WARNED__ that this will authorize any Google account holder to authenticate with OAuth to your cluster services.
   - `GCLOUD_K8S_CLUSTER` specifies the name of the GKE cluster.
   - `GCLOUD_K8S_VERSION` specifies the version of Kubernetes to use for creating the cluster. The [BKPR Kubernetes version support matrix](../README.md#kubernetes-version-support-matrix-for-bkpr-10) lists the base Kubernetes versions supported by BKPR. `gcloud container get-server-config --project ${GCLOUD_PROJECT} --zone ${GCLOUD_REGION}` lists the versions available in your region.
 
-* Create a OAuth Client ID by following these steps
+* Create an OAuth Client ID by following these steps
 
   1. Go to https://console.developers.google.com/apis/credentials
```

Comment:
End this and subsequent sentences with periods.

Reply:
Done